### ISSUE TITLE
Establish relationship with card provider (Visa)

### ISSUE THREAD

**Issue by pavlovcik** [ubiquity/.github#45]

**Comment by pavlovcik** [comment]
@sergfeldman how is this coming along?

**Comment by sergfeldman** [comment]
> @sergfeldman how is this coming along? I haven't heard updates in awhile but this was "assigned" over a month ago.

@pavlovcik Hey, the research is in progress and not yet completed.

I`m posting all the main findings in the comments of the discussion https://github.com/ubiquity/ubiquity-dollar/discussions/307

I think that it will be difficult for us to issue specifically a CREDIT card DIRECTLY from Visa at the moment. 
Having nothing specifically interesting for Visa, I see no reason to contact Visa directly due to the high competition - they stated that 4000+ projects applied to one of the programs in 2022.

**At the same time, there are options for how to do this with the help of intermediaries**
https://github.com/ubiquity/ubiquity-dollar/discussions/307#discussioncomment-5278025

Several points need to be clarified with the intermediaries, first of all
- Possibility of issuing a worldwide card (not only for one of the counties or regions)
- Possibility of a credit limit

**As for the DIRECT cooperation with Visa, I consider "niche product interesting for Visa" as the most promising direction** 

The steps
1. Describe the concept of a niche financial product that is interesting for Visa
(for example, a dedicated card for developers
https://github.com/ubiquity/ubiquity-dollar/discussions/307#discussioncomment-5354018)

2. After that, apply to the programs led by Visa, as well as others 
Program examples
https://partner.visa.com/site/programs/fintech-program.html
https://usa.visa.com/visa-everywhere/everywhere-initiative/initiative.html

It is worth preparing legal and financial data, as there are requirements to be
- A registered corporation in your region
- In good financial standing

Other programs
https://github.com/ubiquity/ubiquity-dollar/discussions/307#discussioncomment-5273514

**Also, there are alternative options for direct cooperation with Visa that can be explored**

For example, cooperation with their wallet
https://github.com/ubiquity/ubiquity-dollar/discussions/307#discussioncomment-5624063

Visa Everywhere Initiative LGBTQ+ edition 
https://usa.visa.com/visa-everywhere/everywhere-initiative/special-edition-competitions.html

**Comment by pavlovcik** [comment]
I understand that credit cards are less common to issue. Let's just do a debit card. What's the status for issuing debit cards?

**Comment by sergfeldman** [comment]
@pavlovcik Yep, issuing a debit card is easier than a credit card.

The status of issuing a debit card is similar to the one described above.
I focused on researching a niche product that Visa or MasterCard might be interested in and ways of collaborating with them.
If Visa or MasterCard accepts us into their program, it will open new opportunities for attracting investors and partners.

But if we just need to issue cards, then let's clarify the high-level requirements:
**- What is our vision**: audience - do we want to issue cards for DevPool developers and for us, or a wider audience - then what kind of audience?
How many cards do we expect to issue and in what time frame? 
What amount of payments will be made via them?
**- Provider:** Visa or MasterCard or Union Pay or any 
Can we consider issuing Visa cards from fintech startups or intermediary companies?
**- Card type:** debit (not prepaid)
**- Base currency:** USD or can be EURO or can be another 
**- Region:** the region of issue (the card can be used at all points supported by the provider, but the region of issue is related to KYC and taxation. At the moment I'm not sure that it is possible to issue a worldwide card)
**- Level:** Basic or elite
### - Digital or plastic
**- Cost:** max cost for the issued card that we are ready to pay 
**- Customization needed for us:** plastic card design, the connection of bonus programs, cards limits, other settings
**- Other requirements:** for example 3ds via sms (which is paid), etc

**Comment by sergfeldman** [comment]
@pavlovcik The situation in simple words 
https://github.com/ubiquity/ubiquity-dollar/discussions/307#discussioncomment-5639012

**Comment by sergfeldman** [comment]
I contacted the 3 card issuer companies through their websites. 
They claimed to respond within 2 business days.

**Comment by sergfeldman** [comment]
### Stripe 

Hi there,

Thank you for reaching out to Stripe. I understand that you wish to issue branded debit cards and I will be happy to help.

Jumping right in, Stripe Issuing allows you to easily create, manage, and scale a commercial card program without any setup fees. Get started in minutes and control every detail of your program, from card design to approving transactions in real time.

You can find more information in the link below:
https://stripe.com/en-ie/issuing

Should you have any account specific questions, I would encourage you to to take an extra authentication step before we can proceed. Our users account security is extremely important to us here at Stripe and as a result we can only give out specific information or take certain actions on a Stripe account if you are definitely authenticated yourself as a user on that account.We are going to need you to take extra authentication steps.

You can navigate to the URL below, login and resend the email to us using the form there: https://support.stripe.com/contact/login

Sending a message using that form while logged in will securely connect your message to the account you’re writing from. Thank you for understanding- we need to be certain that we are communicating with the legitimate account holder for certain issues to ensure the safety of your account information.

If you're having trouble in authenticating your account, kindly send us a screenshot so we can troubleshoot the issue and we will find another way to address your concern.

Rest assured, we will take a deeper look into this matter once you have verified your account with us. I am hoping to hear back from you soon.

**Comment by pavlovcik** [comment]
I understood that we were eligible for Visa's partner program - remember when I gave you my login credentials? I would be interested to invest/work with Stripe or Visa directly over the other options I can think of. 

I also think that beyond just the physical cards, we should offer digital cards that can be used for online/digital phone (NFC/Apple Pay) payments. 

Lastly I would like to see a feature, like how privacy.com does it, where the users have the ability to generate unique card numbers for privacy and money management purposes.

We can have users deposit their Ubiquity Dollars into our (centralized) platform where we can debit their dollars (off chain) to settle card payments. This gives our dollar another utility, for real world spending, independent of the DevPool conveniences we have explicitly discussed in the past.

---

@sergfeldman I would really hope that you can just tell them whatever regarding the information they are asking for because I would want to get our hands on functional cards/APIs for research/iteration purposes. Even if it's just a few physical cards and then access to the APIs for building. 

But for now this is what I think makes sense in terms of vision. I hope that clarifies my perspective.

**Comment by sergfeldman** [comment]
> Do you have any updates @sergfeldman? If you would like to release the bounty back to the DevPool, please comment `/unassign`

@ubiquibot thank you :)

**Comment by sergfeldman** [comment]
> I understood that we were eligible for Visa's partner program - remember when I gave you my login credentials? I would be interested to invest/work with Stripe or Visa directly over the other options I can think of.
> 
> I also think that beyond just the physical cards, we should offer digital cards that can be used for online/digital phone (NFC/Apple Pay) payments.
> 
> Lastly I would like to see a feature, like how privacy.com does it, where the users have the ability to generate unique card numbers for privacy and money management purposes.
> 
> We can have users deposit their Ubiquity Dollars into our (centralized) platform where we can debit their dollars (off chain) to settle card payments. This gives our dollar another utility, for real world spending, independent of the DevPool conveniences we have explicitly discussed in the past.
> 
> @sergfeldman I would really hope that you can just tell them whatever regarding the information they are asking for because I would want to get our hands on functional cards/APIs for research/iteration purposes. Even if it's just a few physical cards and then access to the APIs for building.
> 
> But for now this is what I think makes sense in terms of vision. I hope that clarifies my perspective.

@pavlovcik Thank you for the input!
Potentially, Visa Sandbox can cover this need.
I will get the access and share to you.
Simultaneously, I will continue research on other options.

**Comment by sergfeldman** [comment]
Potentially, we will have a call with [Marqeta](https://www.marqeta.com/) (one of the leading card issuers)

**Comment by pavlovcik** [comment]
> Potentially, we will have a call with [Marqeta](https://www.marqeta.com/) (one of the leading card issuers)
> 
> 
> 
> 

https://t.me/UbiquityDAO/27489/29893

You should be able to handle this without me. I don't want to be on introductory calls. Good luck @sergfeldman

**Comment by sergfeldman** [comment]
> > Potentially, we will have a call with [Marqeta](https://www.marqeta.com/) (one of the leading card issuers)
> 
> https://t.me/UbiquityDAO/27489/29893
> 
> You should be able to handle this without me. I don't want to be on introductory calls. Good luck @sergfeldman

@pavlovcik Okay!
Please confitm your requirements:

**Comment by sergfeldman** [comment]
> ### [ **[ CLAIM 200 DAI ]** ](https://pay.ubq.fi?claim=eyJwZXJtaXQiOnsicGVybWl0dGVkIjp7InRva2VuIjoiMHg2QjE3NTQ3NEU4OTA5NEM0NERhOThiOTU0RWVkZUFDNDk1MjcxZDBGIiwiYW1vdW50IjoiMjAwMDAwMDAwMDAwMDAwMDAwMDAwIn0sIm5vbmNlIjoiMTk0NTUwODM0NTQ3MzE0OTAwMjY3MDIyMTM5Mjg0Mzc3NjQ2OTgyMTIyMjQ0MzcxNzUyOTIxNjU0MDI3MzM2ODI3NTk1MjQ5NjkyNCIsImRlYWRsaW5lIjoiMTE1NzkyMDg5MjM3MzE2MTk1NDIzNTcwOTg1MDA4Njg3OTA3ODUzMjY5OTg0NjY1NjQwNTY0MDM5NDU3NTg0MDA3OTEzMTI5NjM5OTM1In0sInRyYW5zZmVyRGV0YWlscyI6eyJ0byI6IjB4QzNmZEM0ODZFRWE2M0Q3OTYwZTUwQ0M1NDA5ZmJlQTQzNGE2ZkRmMyIsInJlcXVlc3RlZEFtb3VudCI6IjIwMDAwMDAwMDAwMDAwMDAwMDAwMCJ9LCJvd25lciI6IjB4Zjg3Y2E0NTgzQzc5MjIxMmU1MjcyMGQxMjdFN0UwQTM4QjgxOGFEMSIsInNpZ25hdHVyZSI6IjB4MjIxM2FlOWZjNzZkODI1ZjBiMWQ4ZjQxZDRiNWVjYTRiOGNhMzgxODU1ZTg4NGEyMGUwMDI4ZWRhMjgzZjM5Yjc4ZmUyZmEzOWM2NDBjYTk2MDQ3MjFjZDZlNGM5MWU1NDA5YjZhMTRlYTJmOTE5NGFhY2JjYzRjYzMxYmVjNTMxYyJ9)
> `0xC3fdC48...34a6fDf3`

The task was closed not intentionally.

**Comment by pavlovcik** [comment]
> * Provider: Visa
> * Card type: debit (not prepaid)
> * Base currency: USD
> * Region: the region of issue is the US

I need to get my hands on one so can we do Asia?

> * Level: Basic (not elite)
> * Digital (no plastic)
> * Cost: max cost for the each issued card that we are ready to pay is 5 USD per month, issuing should be free

Monthly charge? I'm against that. Are there no alternatives? 

> * Customization needed for us: no customisation
> * Other requirements: no other requirements

I think at minimum I would want a digital card added to my Apple Pay. That way I will be able to use it for most things. A physical card of course will work for everything else but is not 100% necessary for "on-the-field" research.

**Comment by sergfeldman** [comment]
**Region:** Asia, Taiwan is ok, Singapore?, Republic of Korea?, the region where card issuing is not limited to residents only, no KYC cards

**Comment by sergfeldman** [comment]
Ubiquity is building a freelance marketplace embedded to GitHub.
We need cards for the developers-freelancers located accross the world.
The primery use case is cash-out crypto to the debit card and then spending in different countries. 

The vision on the card program:

Card provider: Visa or MasterCard 

Region: can consider 
1) The most suitable region to issue cards to spend in the Republic of Korea or a region where card issuing is not limited to residents only
2) No KYC cards

Base currency: USD

Cost: The lower the better, transaction fees are better than fixed fees

Card type: debit (in the future can be credit)

Card level: Basic (not elite)

Digital card (in the future can be plastic)

Customization needed: generate new card number easily - issue and terminate cards in the style of privacy.com 
https://privacy.com/unique-card-number

Number of cards: from 1 to 20 cards per first 3 month

**Comment by sergfeldman** [comment]
Stripe

Thanks for reaching out and for your interest in Stripe.	
Based on a quick review of your business, we’ve determined that your business model might be included on our Restricted Business list, which would mean we can’t support it.	

To help us determine if we can support your business, we recommend creating and activating your account now so that we can conduct a full review. Going through this process in advance ensures you and your developers don’t spend time integrating Stripe if we’re ultimately unable to support your business.
 
If we need any further details about your business once you’ve set up your account, we’ll reach out to let you know. In the meantime, we’ll be happy to answer any questions you might have. — The Stripe team

**Comment by sergfeldman** [comment]
Will contact
https://www.checkout.com/ - one more time
https://www.intergiro.com/
https://www.chainswipe.com/

Then
https://striga.com/
https://www.hydrogen.tech/
https://www.paysafe.com/
https://www.unlimint.com/
https://optimuscards.com/

Simultaneously will look for an issuer in Republic of Korea

**Comment by pavlovcik** [comment]
> Restricted Business list 

I looked at their [docs](https://support.stripe.com/questions/crypto-supportability-and-availability-by-region). Basically it's case-by-case basis and we're probably going to need a Singapore entity.

**Comment by sergfeldman** [comment]
After long discussions, Stripe support announced that they will inform us of the decision on our case within 10 business days

**Comment by pavlovcik** [comment]
> After long discussions, Stripe support announced that they will inform us of the decision on our case within 10 business days

Curious to Q&A discuss results on a call.

**Comment by sergfeldman** [comment]
Attempts to register the Ubiquity business account have been unsuccessful so far.
At the moment, I can't predict how long it will take to issue cards from the Ubiquity account.
https://github.com/ubiquity/ubiquity-dollar/discussions/307#discussioncomment-5791196

As a temporary solution, I can offer to issue a personal card for expenses in the Republic of Korea.

**Comment by sergfeldman** [comment]
@pavlovcik Issuing a card in the Republic of Korea itself for a non-resident in most cases requires the Alien Resident Card (ARC).

Alternatively, you can consider Wise, Capital One, or other international cards.
https://www.finder.com/travel-money/south-korea
https://wise.com/us/blog/international-prepaid-debit-cards
https://wise.com/us/travel-money/south-korean-won-card

**Comment by pavlovcik** [comment]
> issue a personal card for expenses

This doesn't make sense to me. If a "personal card" is issued, doesn't that mean that it's with an existing bank, and that we are not the provider? 

As long as we are able to access the card issuing APIs, and finance cards (cash out Ubiquity Dollars and then handle fiat liquidity with a Ubiquity fiat bank account) then we can properly begin research according to my vision.

**Comment by sergfeldman** [comment]
### Intergiro

Thank you for reaching out to us. We can only accept entities registered in the European Economic Area, not the UK. If you have an entity in the European Economic Area you can sign up for our waiting list for business banking.

I'd like to clarify that we can only send the cards for company expenses to addresses in the European Economic Area, except for virtual ones which can be issued everywhere. Also, I'd like to clarify that our cards are connected to Euro.

**Comment by sergfeldman** [comment]
> This doesn't make sense to me. If a "personal card" is issued, doesn't that mean that it's with an existing bank, and that we are not the provider?

@pavlovcik Yes, from the Ubiquity cards' point of view, releasing a personal card does not make sense.
I'm just worried that you have a payment card while we are looking at how to issue Ubiquity cards.

**Comment by sergfeldman** [comment]
Simultaneously with "classic" card issuers I'm contacting wallet-as-a-service providers.
https://www.paysafe.com/en/embedded-wallets/
https://youtap.com/products/card-management-and-processing/white-label-digital-wallet-and-card/

**Comment by sergfeldman** [comment]
### Stripe

Unfortunately, our banking-as-a-service and card issuing products don’t support crypto at this time. However, Stripe does support crypto with our payments products: https://stripe.com/use-cases/crypto. 

We are considering crypto for our BaaS products in the future though, so I’ve made a note to reach back out when we expand our product capabilities.

With that being said, in the future if you have a new use case with which you'd like to use Stripe Issuing, we'd recommend directly filling out the sales contact form again as that will get your request in front of a sales team member who'd be able to assist!

**Comment by sergfeldman** [comment]
### Responces from wallet-as-a-service providers whom I asked for help in issuing cards:

**sdk.finance**

Your company can develop the product on top of our Platform. We do not provide card issuing - it's a matter of an agreement of a vendor and integration of it but based on my experience it will be hard for you to find a vendor on this stage. I recommend to start your project without cards and generate some transactions / customers experience. With those numbers you can pitch card issuing vendors and activate such a service in the future. 

**Velmie**

Velmie is a technology company. We help our clients to issue cards with the help of our partners. Due diligence process is pretty strict. Considering the current tightening crypto regulation, our partners do not onboard similar clients.

**Comment by sergfeldman** [comment]
Currently, I'm reaching out to relatevly small non classic card issuers - neobanks and payment services.

https://www.airwallex.com/platform/issuing

https://mercuryo.io/baas/

https://arival.com/product/for-blockchain-and-web3

**Comment by sergfeldman** [comment]
Marqeta refused to start cooperation now but will be glad when we reach the minimum requirements
The details https://github.com/ubiquity/ubiquity-dollar/discussions/307#discussioncomment-5848297

**Comment by pavlovcik** [comment]
That's very unusual that it followed up three times. @0xcodercrane

**Comment by sergfeldman** [comment]
We will have a call with Aspire. 
The topics - business registration in Singapore, card issuing, cashback and Buy Now Pay Later services.

**Comment by sergfeldman** [comment]
The first time I got not negative response:

Airwallex can most likely support you. I just have a few questions if you can help answer them below:
- Are the freelancers 1099s of Ubiquity? 
- Are you planning to pay the freelancers on borderless cards? Or what's the use case for the cards and the freelancers?

**Comment by pavlovcik** [comment]
> Are the freelancers 1099s of Ubiquity?

Pretty sure that 1099 is a US tax code. We're not a US company so technically speaking this is a no. Interpreting what they could mean (are the freelancers contractors of Ubiquity - who submitted their tax information to Ubiquity) we could say that for our beta it is something we could accomodate (just core team for example.) 

Are you planning to pay the freelancers on borderless cards? Or what's the use case for the cards and the freelancers?

For our beta we can make accommodations to make them happy. But yes, the vision is to have Visa cards that could be spent anywhere that Visa is accepted.

**Comment by pavlovcik** [comment]
You should've answered the bot more promptly - seems like its mad at you.

**Comment by sergfeldman** [comment]
Airwallex 
API docs https://www.airwallex.com/docs/api#/Introduction

>Apologies for the slight delay in response. I was waiting on my KYC Onboarding Team to look into your business and reason for using Airwallex - to see if it would be ok. 

Your use case is definitely fine, there's just some scrutiny when it comes to onboarding web3 adjacent companies. Considering you're going to use the platform just for cards though, it should be ok.

**Comment by sergfeldman** [comment]
Aspire
>For crypto related entities, we can only offer USD account. For non-crypto company registered in Singapore, we can offer borderless visa cards.

**Comment by sergfeldman** [comment]
I contacted via emails the banks in British Virgin Islands 
https://nationalbank.vg/
https://www.cibcfcib.com/
https://vg.vpbank.com/
https://www.popular.vi/contact-us/
https://republicbankbvi.com/
https://www.1firstbank.com/vi/
https://www.bankasia.com/

**Comment by sergfeldman** [comment]
After conversation with Unlimint:

They are ready to proceed with our case, but only able to provide Banking as a Service to businesses and individuals registered within the EEA.
